src/toil/utils/toilLaunchCluster.py

```diff
@@ -86,6 +97,9 @@ def main():
         except ImportError:
             raise RuntimeError('The aws extra must be installed to use this provisioner')
         provisioner = AWSProvisioner()
+    elif config.provisioner == 'azure':
+        from toil.provisioners.azure.azureProvisioner import AzureProvisioner
```

Comment:
Should have a try / except ImportError here like the aws one:
`raise RuntimeError('The aws extra must be installed to use this provisioner')`

Reply:
Fixed